src/components/ArtistList.js

```diff
@@ -0,0 +1,45 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+export default ArtistList
+
+ArtistList.propTypes = {
+  artists: PropTypes.array,
+}
+
+function ArtistList({ artists }) {
+  return (
+    <div>
```

Comment:
What's the all enclosing `<div>` used for? I believe it's obsolete as <ListStyled> is only one element.